land_use/models.py

```diff
@@ -0,0 +1,247 @@
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class LandUseTI(models.Model):
```

Comment:
Is it not the same of LandUseClasses?

Reply:
LandUseTI and LandUseClasses are models with different grouping information.
After accessing this APP to the production database we will test performance with LandUseTI.